source/billing.txt

```diff
@@ -84,7 +89,7 @@ request.
 There are several types of requests:
 
 - **Function Executions**, such as when a user calls a function from a client
-  app or when a function calls an external service. If an executing function
+  app, when a function calls an external service, or when a custom error function is invoked by a failed EventBridge trigger. If an executing function
```

Comment:
[nit] we have moved towards calling the error functions `error handlers`. But if it makes more sense to call it a custom error function in this context, that makes sense to me too :) 
```suggestion
app, when a function calls an external service, or when a custom error handler is invoked by a failed EventBridge trigger. If an executing function
```

Reply:
That works! I'll try to stick to `error handlers` going forward so we're consistent with the terminology 👍